zerver/tests/test_bugdown.py

```diff
@@ -892,6 +898,210 @@ def render(msg: Message, content: str) -> str:
         self.assertEqual(render(msg, content), "<p>We have a NOTHINGWORD day today!</p>")
         self.assertEqual(msg.user_ids_with_alert_words, set())
 
+    def test_alert_words_returns_user_ids_with_alert_words(self) -> None:
+        alert_words_for_users = {
+            'hamlet': ['how'], 'cordelia': ['this possible'],
+            'iago': ['hello'], 'prospero': ['hello'],
+            'othello': ['how are you'], 'aaron': ['hey']
+        }  # type: Dict[str, List[str]]
+        user_profiles = {}  # type: Dict[str, UserProfile]
+        user_ids = set()  # type: Set[int]
+        for (username, alert_words) in alert_words_for_users.items():
+            user_profile = self.example_user(username)
+            user_profiles.update({username: user_profile})
+            user_ids.add(user_profile.id)
+            do_set_alert_words(user_profile, alert_words)
+        sender_user_profile = self.example_user('polonius')
+        user_ids.add(sender_user_profile.id)
+        msg = Message(sender=sender_user_profile, sending_client=get_client("test"))
+        realm_alert_words_automaton = get_alert_word_automaton(sender_user_profile.realm)
+
+        def render(msg: Message, content: str) -> str:
+            return render_markdown(msg,
+                                   content,
+                                   realm_alert_words_automaton=realm_alert_words_automaton,
+                                   user_ids = user_ids)
+
+        content = "hello how is this possible how are you doing today"
+        render(msg, content)
+        expected_user_ids = {user_profiles['hamlet'].id, user_profiles['cordelia'].id, user_profiles['iago'].id, user_profiles['prospero'].id, user_profiles['othello'].id}  # type: Set[int]
```

Comment:
Can you line-wrap this?  Our linters don't enforce line-wrapping on tests, because sometimes it's cleaner to have a long line for a URL or something, but this isn't a case of that :)